src/replication/replicator.rst

```diff
@@ -154,98 +203,253 @@ the following order:
 .. code-block:: javascript
 
     {
-        "_id": "doc_A",
-        "source":  "http://myserver.com:5984/foo",
-        "target":  "http://user:pass@localhost:5984/bar"
+        "_id": "my_rep",
+        "source":  "http://user:pass@localhost:5984/foo",
```

Comment:
Same comment as before, would be happier to see source and target not being on the same host.